openhtmltopdf-core/src/main/java/com/openhtmltopdf/render/ListItemPainter.java

```diff
@@ -150,26 +150,36 @@ private static boolean isInVisiblePageArea(RenderingContext c, PageBox page, Str
                (bottomY >= pageTop && bottomY <= pageBottom);
     }
 
-    private static void drawText(RenderingContext c, BlockBox box, IdentValue listStyle) {
+    private static void drawText(RenderingContext c, BlockBox box) {
         MarkerData.TextMarker text = box.getMarkerData().getTextMarker();
         int x = getReferenceX(c, box);
```

Comment:
The way this code was written (before you touched it) makes this very confusing. Could you add a tiny refactor here so the `x` value is never modified after initialization? This initial value is overwritten in the RTL cases, but it is modified with `-=` in the LTR cases. Could you just modify the LTR else blocks such that they are initializing `x` with the `getReferenceX` function inline?